reference/webhooks.v3.yml

```diff
@@ -1,463 +1,642 @@
+          description: Bad request.
+        '401':
+          description: Unauthorized - the v3 Auth client ID or token in the request are not a valid combination for this store.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '404':
+          description: Webhook with specified id doesn't exist.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '422':
+          description: Invalid JSON request body - missing or invalid data.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
       tags:
         - Webhooks
     delete:
+      operationId: deleteAWebhook
+      summary: Delete a Webhook
+      description: 'Deletes a webhook. Only one webhook at a time can be deleted. When a webhook is deleted, it is returned in the response as a `200` OK.'
+      parameters:
+        - $ref: '#/components/parameters/WebhookId'
+        - $ref: '#/components/parameters/Accept'
+        - $ref: '#/components/parameters/Content-Type'
       responses:
         '200':
-          $ref: '#/components/responses/webhook_Resp'
-      summary: Delete a Webhook
-      description: 'Deletes a webhook. Only one webhook at a time can be deleted. When a webhook is deleted, it is returned in the response as a 200 OK.'
-      operationId: deleteAWebhook
+          description: Webhook deleted.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WebhookResponse'
+        '400':
+          description: Bad request.
+        '401':
+          description: Unauthorized - the v3 Auth client ID or token in the request are not a valid combination for this store.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '404':
+          description: Webhook with specified id doesn't exist.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '422':
+          description: Invalid JSON request body - missing or invalid data.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
       tags:
         - Webhooks
+    parameters: []
+  /hooks/admin:
+    get:
+      operationId: getHooksAdmin
+      summary: Get Admin Info
+      description: |-
+        List all notification emails, webhooks and blacklisted domains associated with the API user.
+
+        <div class="HubBlock--callout">
+        <div class="CalloutBlock--info">
+        <div class="HubBlock-content">
+        <!-- type:info -->
+
+        > ### Note
+        > * Webhooks endpoints are available on both the V2 and V3 REST API.

```

Comment:
Do we need to mention the V2 REST API in this documentation?

Reply:
Same as above about waiting to see if we are going to break this into two spec files for `V2` and `V3`.